table/render_tsv.go

```diff
@@ -0,0 +1,73 @@
+package table
+
+import (
+	"fmt"
+	"strings"
+)
+
+func (t *Table) RenderTSV() string {
+	t.initForRender()
+
+	var out strings.Builder
+
+	if t.numColumns > 0 {
+		if t.title != "" {
+			out.WriteString(t.title)
+		}
+
+		if t.autoIndex && len(t.rowsHeader) == 0 {
+			t.tsvRenderRow(&out, t.getAutoIndexColumnIDs(), renderHint{isAutoIndexRow: true, isHeaderRow: true})
+		}
+
+		t.tsvRenderRows(&out, t.rowsHeader, renderHint{isHeaderRow: true})
+		t.tsvRenderRows(&out, t.rows, renderHint{})
+		t.tsvRenderRows(&out, t.rowsFooter, renderHint{isFooterRow: true})
+
+		if t.caption != "" {
+			out.WriteRune('\n')
+			out.WriteString(t.caption)
+		}
+	}
+
+	return t.render(&out)
+}
+
+func (t *Table) tsvRenderRow(out *strings.Builder, row rowStr, hint renderHint) {
+	if out.Len() > 0 {
+		out.WriteRune('\n')
+	}
+
+	for idx, col := range row {
+		if idx == 0 && t.autoIndex {
+			if hint.isRegularRow() {
+				out.WriteString(fmt.Sprint(hint.rowNumber))
+			}
+			out.WriteRune('\t')
+		}
+
+		if idx > 0 {
+			out.WriteRune('\t')
+		}
+
+		if strings.ContainsAny(col, "\t\n\"") || strings.Contains(col, "    ") {
```

Comment:
@jedib0t  It looks like `\t` gets replaced by `"    "` (4 spaces) in [render_init.go#46](https://github.com/jedib0t/go-pretty/blob/main/table/render_init.go#L46)

So, that's why I'm adding `strings.Contains(col, " ")` here

Reply:
Ok I'll have to refactor the original function. This may not be ideal behavior for the end-user.